src/router.rs

```diff
@@ -127,6 +127,12 @@ pub fn build_router(app: &App) -> RouteBuilder {
     // Metrics
     router.get("/api/private/metrics/:kind", C(metrics::prometheus));
 
+    // Crate ownership invitations management in the frontend
+    router.get(
+        "/api/private/crate-owner-invitations",
```

Comment:
hmm, probably should've noticed this earlier: we've been using snake_case for all the URLs so far, instead of kebab-case

Reply:
Ugh, could you send a PR renaming the endpoints?

Reply:
yep, will do :)